src/Iris/Algebra/OFE.lean

```diff
@@ -241,6 +379,7 @@ instance : COFE Unit where
   compl _ := ()
   conv_compl := ⟨⟩
 
+/-- Functors from COFEs to OFEs -/
```

Comment:
These aren't actually functors from COFEs to OFEs, they are profunctors, aka bifunctors which are contravariant on the left, aka functors `COFEᵒᵖ × COFE ⥤ OFE`.